SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -29,9 +30,14 @@ public SonarWebService(IDownloader downloader, string server)
             {
                 throw new ArgumentNullException("server");
             }
-            
+            if (logger == null)
+            {
+                throw new ArgumentNullException("LoggerVerbosity");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method 'SonarWebService.SonarWebService(IDownloader, string, ILogger)' passes 'LoggerVerbosity' as the 'paramName' argument to a 'ArgumentNullException' constructor. Replace this argument with one of the method's parameter names. Note that the provided parameter name should have the exact casing as declared on the method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AInstantiateArgumentExceptionsCorrectly)